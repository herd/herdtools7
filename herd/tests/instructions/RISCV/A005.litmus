RISCV A005

(* Test NOP instruction*)
{}

P0;

NOP;

forall 0:a0=0
