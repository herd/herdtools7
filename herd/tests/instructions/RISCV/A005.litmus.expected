Test A005 Required
States 1
0:x10=0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:x10=0)
Observation A005 Always 1 0
Hash=7dbd8e4fdafc2b299a8e4b02a379dcaa

